src/components/AppHeader.vue

```diff
@@ -1,56 +1,72 @@
 <template>
-	<!-- TODO: Update to ANTD menu and header -->
-	<div class="header" id="header">
-		<a-row class="veritcal_center">
-			<a-col :span="20">
-				<a-row class="veritcal_center">
-					<a-col :span="4">
-						<h1>SigmaDao</h1>
-					</a-col>
-					<a-col :span="8">
-						<div class="menu_id">
-							<label for="dao_id" style="padding-right: 10px">DAO ID</label>
-							<input id="dao_id" v-model="id" type="number" />
-						</div>
-					</a-col>
-					<a-col>
-						<div>
-							<router-link :to="{ path: '/vote' }">
+	<div class="components-page-header-demo-responsive header">
+		<a-page-header>
+			<template #title>
+				<img src="../assets/logo.png" class="logo" />
+			</template>
+			<template #extra>
+				<WalletConnect />
+			</template>
+			<div class="content">
+				<div class="main">
+					<a-row align="middle" justify="center">
+						<a-col class="menu" :span="24">
+							<div>
+								<!-- add route after page component is added  -->
 								<a-button
 									class="menu_option"
-									:type="
-										currentPageKey === NavigationKey.VOTE ? 'link' : 'text'
-									"
-									@click="() => handleMenuClick(NavigationKey.VOTE)"
-									>Vote</a-button
+									:type="isLinkActive(NavigationKey.CREATE_DAO)"
+									@click="() => handleMenuClick(NavigationKey.CREATE_DAO)"
+									>Create DAO</a-button
 								>
-							</router-link>
-							<router-link :to="{ path: '/addProposal' }">
+
 								<a-button
 									class="menu_option"
-									:type="
-										currentPageKey === NavigationKey.ADD_PROPOSAL
-											? 'link'
-											: 'text'
-									"
-									@click="() => handleMenuClick(NavigationKey.ADD_PROPOSAL)"
-									>Add Proposal</a-button
+									:type="isLinkActive(NavigationKey.ALL_DAO)"
+									@click="() => handleMenuClick(NavigationKey.ALL_DAO)"
+									>All DAOs</a-button
+								>
+								<router-link :to="{ path: '/vote' }">
+									<a-button
+										class="menu_option"
+										:type="isLinkActive(NavigationKey.VOTE)"
+										@click="() => handleMenuClick(NavigationKey.VOTE)"
+										>Vote</a-button
+									>
+								</router-link>
+								<router-link :to="{ path: '/addProposal' }">
```

Comment:
We can use enum here. If in future if we decide to change the name of the endpoint then there will be very less code change.

e.g
enum Pagination {
ADD_PROPOSAL: '/addProposal',
VOTE: '/vote'
}

Reply:
nice suggestion. will update